includes/polling/sensors/state/arubaos-cx.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$stateLookupTable = [
+    // arubaWiredVsfv2OperStatus
+    'no_split' => 0,
+    'fragment_active' => 1,
+    'fragment_inactive' => 2,
+
+    //arubaWiredVsfv2MemberTable
+    'not_present' => 10,
+    'booting' => 11,
+    'ready' => 12,
+    'version_mismatch' => 13,
+    'communication_failure' => 14,
+    'in_other_fragment' => 15,
+];
+
+$sensor_value = $stateLookupTable[$sensor_value];
```

Comment:
You should here ensure that you are on those 2 OIDs only. Cause if somebody adds another state sensor, this code will break the returned value.

Reply:
Agreed. I've added an if to check if we're dealing with the VSF sensors.